frontend/pages/About.js

```diff
@@ -149,6 +149,18 @@ const studentMembers = [
     {
         name: "Vicky Zheng",
         photoSrc: Zheng_Vicky
+    },
+    { 
+        name: "Cindy Zheng",
+        photoSrc: Zheng_Vicky
+    }
+    { 
+        name: "Lisa Li-Liang",
+        photoSrc: Zheng_Vicky
+    }
+    {
+        name: "Kingston Lew",
+        photoSrc: Lew_Kingston
```

Comment:
This block is breaking the JS linter, make sure you include commas after each JSON object.